### PR TITLE
Add notes and effect for Red-Gold Mortality

### PR DESCRIPTION
**Pull request by pedrogrullada**

### CODE REVIEW

File: static/lang/re-en.json [hunk 1]
```diff
@@ -3339,6 +3339,11 @@
                     },
                     "Single": "Single ikon with multiple copies."
                 },
+                "RedGoldMortality": {
+                    "DamageNote": "When you successfully damage an enemy with the ikon, a marking appears around the wound, painted in the red of mortal blood and the gold of divine ichor. When the target would regain Hit Points, it must attempt a Will save against your class DC to determine the effects, and then the marking fades. The marking otherwise fades after 1 minute. @UUID[Compendium.pf2e.feat-effects.Item.TpcfUI2y6Shg63V0]{Effect: Red-Gold Mortality}",
```
Comment (simonward) [inline static/lang/re-en.json:3343]:
```suggestion
                    "DamageNote": "When the target would regain Hit Points, it must attempt a Will save against your class DC to determine the effects. @UUID[Compendium.pf2e.feat-effects.Item.TpcfUI2y6Shg63V0]{Effect: Red-Gold Mortality}",
```

File: static/lang/re-en.json [hunk 2]
```diff
@@ -3339,6 +3339,11 @@
...
+                    "HealingNote": "<p>When the target would regain Hit Points, such as from a healing effect or an ability like fast healing or regeneration, it must attempt a @Check[will|traits:exemplar,ikon,void] save against your class DC to determine the effects, and then the marking fades. The marking otherwise fades after 1 minute.</p>\n<hr>\n<p><strong>Success</strong> The creature regains the full number of Hit Points that would be healed.</p>\n<p><strong>Failure</strong> The creature regains only half the number of Hit Points as the contradictory energies swirl within it.</p>\n<p><strong>Critical Failure</strong> The creature doesn't regain any Hit Points.</p>",
```
Comment (simonward) [inline static/lang/re-en.json:3344]:
```suggestion
                    "HealingNote": "<p>When the target would regain Hit Points, it must attempt a @Check[will|traits:exemplar,ikon,void] save against your class DC to determine the effects.",
```

File: packs/feats/red-gold-mortality.json [hunk 3]
```diff
@@ -30,6 +30,16 @@
...
+            },
+            {
+                "key": "Note",
+                "predicate": [
+                    "divine-spark:{item|flags.pf2e.rulesSelections.ikon}",
+                    "item:tag:physical-ikon:{item|flags.pf2e.rulesSelections.ikon}"
+                ],
+                "selector": "strike-damage",
+                "text": "PF2E.SpecificRule.Exemplar.RedGoldMortality.DamageNote",
+                "title": "PF2E.SpecificRule.Exemplar.RedGoldMortality.Label"
```
Comment (simonward) [inline packs/feats/red-gold-mortality.json:42]: Would `"title": "{item|name}"` work here?

File: packs/feat-effects/effect-red-gold-mortality.json [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+            "value": 1
+        },
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder War of Immortals"
+        },
+        "rules": [
+            {
+                "key": "Note",
```
Comment (simonward) [inline packs/feat-effects/effect-red-gold-mortality.json:25]: This is somewhat awkward, as the Note will only appear once the healing has been applied. Don't think there's an alternate way though.

Comment (pedrogrullada) [reply packs/feat-effects/effect-red-gold-mortality.json:25]: Yeah, it's the only way if we're gonna do it at all. Healing is easy to undo, at least.

File: packs/feats/red-gold-mortality.json (hunk 3 above)
Comment (pedrogrullada) [reply packs/feats/red-gold-mortality.json:42]: Not quite, the feat name has the Ikon name from the Choice Set, which I left purposefully as it's important to know to which ikon an ikon feat is associated.

File: static/lang/re-en.json (hunk 2 above)
Comment (pedrogrullada) [reply static/lang/re-en.json:3344]: Without the results there isn't much point to this Note

File: static/lang/re-en.json [hunk 5]
```diff
@@ -3339,6 +3339,11 @@
                     },
                     "Single": "Single ikon with multiple copies."
                 },
+                "RedGoldMortality": {
+                    "DamageNote": "When the target would regain Hit Points, it must attempt a Will save against your class DC to determine the effects. @UUID[Compendium.pf2e.feat-effects.Item.TpcfUI2y6Shg63V0]{Effect: Red-Gold Mortality}",
```
Comment (pedrogrullada) [inline static/lang/re-en.json:3343]:
```suggestion
                    "DamageNote": "If you successfully damage the enemy, when the target would regain Hit Points, it must attempt a Will save against your class DC to determine the effects. @UUID[Compendium.pf2e.feat-effects.Item.TpcfUI2y6Shg63V0]{Effect: Red-Gold Mortality}",
```
We actually do need the damage bit in there, as an enemy could be immune to the damage and then this wouldn't take effect.

File: packs/feat-effects/effect-red-gold-mortality.json (hunk 4 above)
Comment (pedrogrullada) [reply packs/feat-effects/effect-red-gold-mortality.json:25]: Eh, thinking about it, this is unacceptably clunky. We could do a note on just the damage roll with the ikon, but that seems a bit useless as there would be nothing actionable on that note. I'll probably just close this later if no better alternative comes up.

Comment (simonward) [reply packs/feat-effects/effect-red-gold-mortality.json:25]: Yeah. I like the idea, but might want to hold off until better automation options exist?

**Pull request closed without merging**